tests/test_generate_success.py

```diff
@@ -0,0 +1,57 @@
+# Global imports
+import pytest
+from gatorgrade.generate.generate import generate_config
+
+"""Test for best case scenario"""
+
+
+@pytest.fixture()
+def setup(tmp_path):
+    """Setup directory for tests"""
+    # Given this file structure
+    src_dir = tmp_path / "src"
+    src_dir.mkdir()
+    python_file = src_dir / "test.py"
+    python_file.write_text("import sys")
+    writing_dir = tmp_path / "writing"
+    writing_dir.mkdir()
+    reflection = writing_dir / "reflection.md"
+    reflection.write_text("# Reflection")
+    yield tmp_path
+
+
+def test_generate_config_creates_gatorgrade_yml(setup):
+    """Test to see that gatorgrade_yml file exist in file structure"""
+    # When generate_config is called
+    generate_config(["src"], setup)
```

Comment:
I am not so sure, but the second argument is supposed to be a string in the implementation function. Can you double-check the `setup` refers to the fixture function or the repo name?